src/main/java/com/mraof/minestuck/data/loot_table/MSMiscLootTables.java

```diff
@@ -32,5 +35,20 @@ public void accept(BiConsumer<ResourceLocation, LootTable.Builder> lootProcessor
 						.add(LootItem.lootTableItem(Items.DEAD_HORN_CORAL).setWeight(10).setQuality(0))
 				)
 		);
+		
+		lootProcessor.accept(MSLootTables.LOTUS_FLOWER_DEFAULT, LootTable.lootTable()
+				.withPool(LootPool.lootPool().name("computer").setRolls(ConstantValue.exactly(1))
+						.add(LootItem.lootTableItem(MSItems.COMPUTER_PARTS.get()).setWeight(10).setQuality(3)))
+				.withPool(LootPool.lootPool().name("code").setRolls(ConstantValue.exactly(1))
+						.add(LootItem.lootTableItem(MSItems.SBURB_CODE.get()).setWeight(10).setQuality(3)).apply(SetNbtFunction.setTag(giveParadoxFragment())))
+		);
+	}
+	
+	private static CompoundTag giveParadoxFragment()
+	{
+		CompoundTag nbt = new CompoundTag();
+		nbt.putBoolean("hasParadoxInfo", true);
```

Comment:
Probably better to handle this in `IncompleteSburbCodeItem`, so that *how* the paradox fragment is stored in nbt is determined all in one place.

Reply:
Moved it into IncompleteSburbCodeItem. Didnt seem appropriate to combine it with the existing methods if that was part of the recommendation so let me know if you want further changes